README.md

```diff
@@ -1,4 +1,4 @@
-# Dependency Injection Graph
+# dig - A dependency injection framework for Go
```

Comment:
Might even makes sense to do:

# go.uber.org/dig - A dependency injection framework for Go